protobuf.bzl

```diff
@@ -141,10 +142,12 @@ def cc_proto_library(
         deps=[s + "_genproto" for s in deps],
         includes=includes,
         protoc=protoc,
+        visibility=visibility,
```

Comment:
Hmm, this one is tricky. The _proto_gen needs to be depended on by other _proto_gen which will have a "_genproto" in the suffix. Simply propagate the visibility here might not give the visibility to those _proto_gen targets, if the visibility attribute contains specific cc_proto_libary rules.

I don't know what's the best option, probably mark this always public and restrict only on the cc_library?